source/shared/cpp/ObjectModel/Enums.cpp

```diff
@@ -21,7 +21,7 @@ namespace AdaptiveSharedNamespace
             {AdaptiveCardSchemaKey::BackgroundImage, "backgroundImage"},
             {AdaptiveCardSchemaKey::BackgroundImageUrl, "backgroundImageUrl"},
             {AdaptiveCardSchemaKey::BaseCardElement, "baseCardElement"},
-            {AdaptiveCardSchemaKey::Bleed, "bleed"},
+            //{AdaptiveCardSchemaKey::Bleed, "bleed"},
```

Comment:
>//{AdaptiveCardSchemaKey::Bleed, "bleed"}, [](start = 12, length = 42)

What's this supposed to be? :)

Reply:
thanks, that shouldn't be there

---
In reply to: [266114463](https://github.com/Microsoft/AdaptiveCards/pull/2544#discussion_r266114463) [](ancestors = 266114463)

Reply:
cool :)

---
In reply to: [266114743](https://github.com/Microsoft/AdaptiveCards/pull/2544#discussion_r266114743) [](ancestors = 266114743,266114463)